TestApi/Model/TestCountryRepository.cs

```diff
@@ -0,0 +1,104 @@
+using Xunit;
```

Comment:
Looks good for the most part, but do we not want to test that correct data and points are included and also when the operation is not successful?